builds/e2e/nested-longhaul.yaml

```diff
@@ -17,209 +17,35 @@ stages:
-          build.repo.path: '$(Build.SourcesDirectory)'
-          edgelet.source.branch: '$(edgelet.package.branchName)'
-          images.source.branch: '$(images.branchName)'
-          edgelet.artifact.name: '$(edgelet.artifact.name)'
-          images.artifact.name: '$(az.pipeline.images.artifacts)'
-          container.registry: '$(cr.address)'
-          container.registry.username: '$(edgebuilds-azurecr-io-username)'
-          container.registry.password: '$(edgebuilds-azurecr-io-pwd)'
-          iotHub.connectionString: '$(IotHubStressConnString)'
-          eventHub.connectionString: '$(EventHubStressConnStr)'
-          loadGen.message.frequency: '$(loadGen.message.frequency)'
-          longHaul.desiredModulesToRestartCSV: '$(longHaul.desiredModulesToRestartCSV)'
-          longHaul.restartIntervalInMins: '$(longHaul.restartIntervalInMins)'
-          longHaul.sendReportFrequency: '$(longHaul.sendReportFrequency)'
-          test.runtimeLogLevel: '$(test.runtimeLogLevel)'
-          logAnalytics.workspaceId: '$(kvLogAnalyticWorkspaceId)'
-          logAnalytics.sharedKey: '$(kvLogAnalyticSharedKey)'
-          twinTester.twinUpdateSize: '$(twinTester.twinUpdateSize)'
-          twinTester.twinUpdateFrequency: '$(twinTester.twinUpdateFrequency)'
-          twinTester.twinUpdateFailureThreshold: '$(twinTester.twinUpdateFailureThreshold)'
-          twinTester.edgeHubRestartFailureTolerance: '$(twinTester.edgeHubRestartFailureTolerance)'
-          metricsCollector.metricsEndpointsCSV: '$(metricsCollector.metricsEndpointsCSV)'
-          metricsCollector.scrapeFrequencyInSecs: '$(metricsCollector.scrapeFrequencyInSecs)'
-          metricsCollector.uploadTarget: '$(metricsCollector.uploadTarget)'
-          metricsCollector.hostPlatform: '$(hostPlatform)'
-          longHaul.parentHostname: '$(parentName)'
-          longHaul.parentEdgeDevice: '$(parentDeviceId)'
-          testResultCoordinator.storageAccountConnectionString: '$(EdgeLonghaulStorageAccountConnString)'
-          quickstart.package.name: '$(quickstart.package.name)'
-          aziotis.artifact.name: '$(aziotis.artifact.name)'
-          aziotis.package.filter: '$(aziotis.package.filter)'
+    parameters:
+      transportType: amqp
+  - template: templates/nested-longhaul-deploy-amd64.yaml
+    parameters:
+      transportType: amqp
+  - template: templates/nested-parent-vm-setup.yaml
+    parameters:
+      transportType: mqtt
+  - template: templates/nested-longhaul-deploy-amd64.yaml
+    parameters:
+      transportType: mqtt
+
   - job:  Clean_up      
     dependsOn:
-      - SetupVM_level5 
-      - SetupVM_level4
-      - Deploy_Longhaul_Linux_Amd64
-      - Deploy_Longhaul_Linux_Arm32
-      - Deploy_Longhaul_Linux_Arm64
+      - SetupVM_level5_amqp
+      - SetupVM_level5_mqtt
+      - SetupVM_level4_amqp
+      - SetupVM_level4_mqtt
+      - Deploy_Longhaul_Linux_Amd64_amqp
+      - Deploy_Longhaul_Linux_Amd64_mqtt
```

Comment:
Just for my understanding, do these account for level 3? I see level 3 in our pool below but not as a depends, so just curious about that :)

Reply:
"Deploy_longhaul_linux_amd64_[amqp/mqtt]" are the names for the level 3 deployment. I'll change the name to reflect that